.travis.yml

```diff
@@ -3,5 +3,8 @@ language: ruby
 cache: bundler
 script:
   - docker-compose -f travis-docker-compose.yml run web tox
+  - docker-compose -f travis-docker-compose.yml run watch npm install
```

Comment:
I have `node_env='production'` in `travis-docker-compose.yml`, which we had before. Should we not be setting that?